index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rizwan Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-size: cover;
+    }     
+    </style>
+</head>
+<body>
+
+    <header> 
+
+        <div class="nav-menu-content">
+
+            <div class="logo"><a href="#" class=mylogo> My Logo </a></div>
```

Comment:
- [ ] Nice job 👏  👏   making the logo a `<a>` tag however you still need to work on its style while removing the underline.

Reply:
Thank you